src/tailwind.js

```diff
@@ -1,86 +1,74 @@
-module.exports = {
+const _ = require("lodash"); //eslint-disable-line @typescript-eslint/no-var-requires
+const commonUiConfig = require("@govtechsg/tradetrust-ui-components/build/tailwind"); //eslint-disable-line @typescript-eslint/no-var-requires
+
+// https://tailwindcss.com/docs/theme
+const localConfig = {
   // future: {
   //   removeDeprecatedGapUtilities: true,
   //   purgeLayersByDefault: true,
   // },
   // purge: ["./src/**/*.ts", "./src/**/*.tsx"], // not to purge, cos common ui might need certain styles (should purge specific layers instead if really need)
   theme: {
-    container: (theme) => ({
-      center: true,
-      padding: theme("spacing.4"),
-    }),
     fontFamily: {
-      "source-sans-pro": ["Source Sans Pro", "sans-serif"],
-      montserrat: ["Montserrat", "sans-serif"],
+      sans: ["Roboto", "sans-serif"],
+      display: ["Roboto", "sans-serif"],
+      body: ["Roboto", "sans-serif"],
     },
```

Comment:
^ this is correct/better i think

we should release a patch at commonui side to NOT include font family at all in the exported tw config (we can still defined the family in the root tw config), then let application end to decide for themselves like how this is done now

Reply:
ok